Tasks/PublishPipelineArtifactV1/task.json

```diff
@@ -9,7 +9,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 0,
+        "Minor": 1,
         "Patch": 0
```

Comment:
For some reason, github is not allowing me to comment on line #18 🤔 . Anyhow, have to change the agent version once those changes go in.

Reply:
yes, minAgentVersion will be updated only after the agent is released.